ros_buildfarm/templates/release/binarydeb_job.xml.em

```diff
@@ -9,7 +9,7 @@ but disabled since the package is blacklisted (or not whitelisted) in the config
   <properties>
 @(SNIPPET(
     'property_log-rotator',
-    days_to_keep=180,
+    days_to_keep=730,
```

Comment:
Was there a reason for this to be shorter than the other ones ?

Reply:
Which ones are you referring to with "the other ones"? It had and has the same value as the corresponding sourcedeb job.

Reply:
That was not clearly formulated: was there a reason for source jobs and binary jobs to have shorter times than the other jobs updated in this PR?

Reply:
Not really. Some job types have even less days. For each type we are just "eyeballing" what might make sense :stuck_out_tongue_winking_eye:

Reply:
sounds good, thanks